minix/commands/swifi/systest.c

```diff
@@ -73,7 +73,7 @@ main(int argc, char * argv[])
     goto Cleanup;
   }
 
-  memset(res, 0, sizeof(res));
+  memset(res, 0, sizeof(*res));
```

Comment:
I'm afraid that's not correct.. 'res' is a pointer, not an array, so you have to supply the size of what's being pointed to - in this case, "arg * sizeof(swifi_result_t)".